src/Lib/Canvas.vala

```diff
@@ -24,14 +24,13 @@
 
 public class Akira.Lib.Canvas : Goo.Canvas {
     public weak Akira.Window window { get; construct; }
-
```

Comment:
Don't randomly remove empty lines between attribute types, those are there for a reason.